src/docs/User/updateProfile.yaml

```diff
@@ -0,0 +1,112 @@
+paths:
+  /api/user/profile/update:
+    patch:
+      tags:
+        - User
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                firstName:
+                  type: string,
+                lastName:
+                  type: string,
+                phoneNumber:
+                  type: string,
+                age:
+                  type: integer,
+                gender:
+                  type: string,
+                nationality:
+                  type: string,
+                refugee:
+                  type: boolean,
+            examples:
+              'CORRECT INFO SENT FOR UPDATE':
+                value: '{
+                  "firstName" : "Nilay",
+                  "lastName": "Aydin",
+                  "phoneNumber": "+679-123456789012",
+                  "age": 25,
+                  "gender": "Female",
+                  "nationality": "Turkey",
+                  "refugee": false,
+                  "address":{"country":"Germany"},
+                  }'
+              'PHONENUMBER NOT INCORRECT FORMAT':
+                value: '{
+                  "phoneNumber": "+60"
+                  }'
+              'AGE IS NOT A NUMBER':
```

Comment:
I had specified [before](https://github.com/RCDD-202203-TUR-BEW/backend-capstone-turkey-lets-share/pull/40#discussion_r945130675): Mongoose does accept a string as an integer in some cases. You can change the 25 to `25e` maybe, or a 3-digit number.
![Screenshot from 2022-08-16 16-17-15](https://user-images.githubusercontent.com/102853969/184893781-c0d7691a-8981-4f05-815a-73089e4d592e.png)

Reply:
got it it's set as a number in mongoose at the end so no worries